scripts/generateTestData.js

```diff
@@ -2,14 +2,16 @@ const fs = require('fs');
 
 import {countries, vaccines, touchstones, activityTypes} from "../src/Data.ts"
 
-fs.writeFile("data/test/impactData.js", generateData(), function (err) {
+for (let i in touchstones) {
+    var fileName = "data/test/impactData_" + touchstones[i] + ".js";
+    fs.writeFile(fileName, generateData(), function (err) {
     if (err) {
         return console.log(err);
     }
 
-    console.log("Fake data saved to data/test/impactData.js")
+    console.log("Fake data saved to " + fileName)
```

Comment:
So `generateData` should now take the touchstone name as an argument and do away with the mapping step from all touchstones (starting on line 23)